lib/evaluate.js

```diff
@@ -279,6 +296,108 @@ const calculateInetGroupSuccessRates = (taskGroups) => {
   return result
 }
 
+/**
+ * @param {import('./typings').Measurement[]} measurements
+ * @param {import('./typings').Point} telemetryPoint
+ */
+const reportRetrievalStats = (measurements, telemetryPoint) => {
+  const totalCount = measurements.length
+  if (totalCount < 1) {
+    telemetryPoint.intField('measurements', totalCount)
```

Comment:
```suggestion
    telemetryPoint.intField('measurements', 0)
```

for consistency (or change line below)